src/core/thread/mle.cpp

```diff
@@ -1560,6 +1560,7 @@ ThreadError Mle::SendChildUpdateRequest(void)
     VerifyOrExit((message = NewMleMessage()) != NULL, ;);
     SuccessOrExit(error = AppendHeader(*message, Header::kCommandChildUpdateRequest));
     SuccessOrExit(error = AppendMode(*message, mDeviceMode));
+    SuccessOrExit(error = AppendSourceAddress(*message));
```

Comment:
Yes, it is a known issue in internal V39 which has been fixed in internal V40 Test Harness.